src/lib.rs

```diff
@@ -294,6 +294,8 @@ impl BitcoinD {
             p2p_args,
             conf_args
         );
+
+        #[allow(clippy::needless_borrow)] // would break 1.41.1 if fixed
```

Comment:
FYI you can configure clippy to know about your MSRV and not suggest wrong things.